nymea-plugins-modbus.pro

```diff
@@ -46,6 +47,9 @@ QMAKE_EXTRA_TARGETS += lrelease
 # For Qt-Creator's code model: Add CPATH to INCLUDEPATH explicitly
 INCLUDEPATH += $$(CPATH)
 
+CONFIG += selection
+PLUGINS=huawei
```

Comment:
to be removed

Reply:
you can set this in qtcreator ;)

Also, would be nice if you could align this to the other plugin repos which use "WITH_PLUGINS" and "WITHOUT_PLUGINS" to cherry-pick built plugins in a single argument.

Reply:
Used for crosscompile...